oat-sa/extension-tao-testqti-previewer
# Feature/CON-24/Test Preview the topBlock component

Related to: https://oat-sa.atlassian.net/browse/CON-24
 
The topBlock component is used in Test Preview with new UI test runner.

How to test:
- go to Test Preview with nextGen testRunner 
  check `config/tao/client_lib_config_registry.conf.php` 
```
       'taoTests/controller/tests/action' => array(
            'provider' => 'qtiTestNUI'
        ),
        'taoQtiTest/controller/creator/creator' => array(
            'provider' => 'qtiTestNUI'
        ),
```
- check topBlock component
![image](https://user-images.githubusercontent.com/25976342/97267303-fd31c800-183a-11eb-9292-f9580db85840.png)
